solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -621,13 +635,60 @@ contract RandomBeaconGovernance is Ownable {
             groupCreationFrequency,
             groupLifetime,
             newDkgResultChallengePeriodLength,
+            dkgResultChallengeExtraGas,
             dkgResultSubmissionTimeout,
             dkgSubmitterPrecedencePeriodLength
         );
         dkgResultChallengePeriodLengthChangeInitiated = 0;
         newDkgResultChallengePeriodLength = 0;
     }
 
+    /// @notice Begins the DKG result challenge extra gas update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newDkgResultChallengeExtraGas New DKG result challenge extra gas
```

Comment:
Missing dot at the end of parameter description.

Reply:
The problem with `RandomBeacon`, `WalletRegistry`, and libraries is that they were never updated to use the new convention. Most of the functions use parameter description with no dots, some others use dots (minority). If we want to fix it, then I would prefer to do it in a separate PR.